assets/js/spec/game_test.js

```diff
@@ -34,20 +38,25 @@ describe('New game', function() {
     });
     afterEach(function() {jasmine.Ajax.uninstall()});
 
-    it('creates buttons based on JSON response', function() {
-        testButtonSetup(3, "12345");
+    it('creates buttons and data based on JSON response', function() {
+        testButtonSetup(3, PID, GID);
         expect($('[data-position]').length).toBe(9);
-        testButtonSetup(5, "12345");
+        testButtonSetup(5, PID, GID);
         expect($('[data-position]').length).toBe(25);
+
+        expect(game.gameId).toBe(GID);
+        expect(game.playerId).toBe(PID);
+        expect(game.mark).toBe('X');
     });
 
     it('creates buttons with proper links', function() {
-        testButtonSetup(3, "12345");
+        testButtonSetup(3, PID, GID);
         $('[data-position=1]').trigger('click');
         var mostRecent = requests.mostRecent();
         expect(mostRecent.url).toMatch(/^move\?/);
         expect(mostRecent.url).toMatch(/position=1/);
         expect(mostRecent.url).toMatch(/player-id=12345/);
+        expect(mostRecent.url).toMatch(/game-id=77777/);
```

Comment:
If you're going to use file level constants for game/player id then it's better to interpolate them in the matchers here